### DISCUSSION TITLE
Cursor problems with HDMI Virtual Display

### DISCUSSION THREAD

**Discussion by DerekL0576** [rustdesk/rustdesk#4459] [General]
### Bug Description

I have a strange problem with the cursor not changing when hovering over splitters or resizing windows etc.

I have 4 PCs where everything works perfectly.  This one has an HDMI virtual display adapter because I use it as a headless server and need the adapter to enable hardware transcoding for Plex etc.

When I install RustDesk, it works fine until I reboot the remote machine. Then the cursor doesn't work as it should.  

A workaround I found is to connect with RDP with the Windows RDP client, then log in again on the RustDesk session. The cursor works again until the next reboot.

I'm not sure what's happening to make it start working.  I've tried a number of other things like restarting the RustDesk service but only the RDP trick works. Even updating RustDesk has no effect.

Also, the cursor is a blue circle and only changes to the correct cursor when I click on the areas where it _should_ actually change on mouse over. eg. It'll change to the resize cursor when clicking on the window frame.  On 1.1.9, nothing happens at all when I click the area, 1.2.0 I can actually still resize the window, there's just no cursor change to tell where the area is before you click it.

### How to Reproduce

Using a PC with an HDMI Virtual Display adapter, reboot the remote machine and try to connect again.  The cursor doesn't change like it's supposed to.

### Expected Behavior

The cursor should change as it moves over resizable areas like splitter etc.

### Operating system(s) on local side and remote side

Window 11

### RustDesk Version(s) on local side and remote side

1.1.9 - 1.2.0 nightly

### Screenshots

![rustdesk-cursor](https://github.com/rustdesk/rustdesk/assets/3249612/214c0f6b-d150-4196-86ef-54333c0d759e)

**Comment by DerekL0576** [comment]
In what way is this a feature request?  It could be argued that it would be an enhancement to not have this bug, but it IS a bug that occurs under conditions that you haven't tested. It's never going to get the right attention in here so whatever.

**Comment by jan-the-maker** [comment]
I am also experiencing this problem. I think I found a way to reliably get the cursor back to show up but I still need to verify its validity. I think having another session in view only (not sure its the view only mode OR the fact that' its another session/IP address) mode helps - just switching to it seems to bring the cursors back for all rust desk sessions. Anyway the problem is real but there seems to be a way to get them back - although not optimal it's possible. I will update here once I have found the optimal way.

**Comment by masterwebsk** [comment]
any update to this issue ? we switched real monitor to hdmi dummy plug and got the same blue circle instead to cursors in windows. there is problem with resizing windows and other cases. Interesting is, that TeamViewer is showing cursors as usually.
